Modules/_sqlite/connection.c

```diff
@@ -293,6 +298,10 @@ int pysqlite_connection_register_cursor(pysqlite_Connection* connection, PyObjec
 
 PyObject* pysqlite_connection_cursor(pysqlite_Connection* self, PyObject* args, PyObject* kwargs)
 {
+        if (self == NULL) {
+                return NULL;
+        }
+        
```

Comment:
Indentation is off, and there is trailing whitespace. Ref. PEP 7.

```suggestion
    if (self == NULL) {
        return NULL;
    }

```